setup.py

```diff
@@ -8,7 +8,14 @@
 from IPython.html.nbextensions import install_nbextension
 from IPython.html.services.config import ConfigManager
 
-VERSION_FILE = os.path.join(os.path.dirname(__file__), 'VERSION')
+# Get location of this file at runtime
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+# Eval the version tuple and string from the source
+VERSION_NS = {}
+with open(os.path.join(HERE, 'urth/cms/_version.py')) as f:
```

Comment:
If you use os.path.join, then it makes sense to split it completely:
`open(os.path.join(HERE, 'urth', 'cms', '_version.py'))`